packages/paste-core/components/timeline/src/Timeline.tsx

```diff
@@ -0,0 +1,77 @@
+import { Box, type BoxStyleProps } from "@twilio-paste/box";
+import { css, styled } from "@twilio-paste/styling-library";
+import React from "react";
+
+import { TimelineContext } from "./TimelineContext";
+import type { Orientation, TimelineProps } from "./types";
+
+const VariantStyles: {
+  [key in Orientation]: BoxStyleProps;
```

Comment:
praise: love the type safety here

Reply:
+1!! but we probably don't need VariantStyles anymore if we're not doing horizontal right? same with the keys in ItemSeparatorStyles